ydb/core/tx/schemeshard/schemeshard__operation.cpp

```diff
@@ -183,7 +191,7 @@ THolder<TProposeResponse> TSchemeShard::IgniteOperation(TProposeRequest& request
                                     << ", already accepted parts: " << operation->Parts.size()
                                     << ", propose result status: " << NKikimrScheme::EStatus_Name(response->Record.GetStatus())
                                     << ", with reason: " << response->Record.GetReason()
-                                    << ", tx message: " << GetRecordForPrint(record).ShortDebugString());
```

Comment:
Old method masked UserToken. Do it masked with new method?

Reply:
Yes